import_musicbrainz_row_ids.py

```diff
@@ -0,0 +1,115 @@
+from listenbrainz import db
+from brainzutils import musicbrainz_db
+from listenbrainz.webserver import create_app
+from listenbrainz.webserver.views.api_tools import publish_data_to_queue
+from listenbrainz.listenstore import InfluxListenStore
+from listenbrainz.webserver.influx_connection import init_influx_connection
+from werkzeug.exceptions import NotFound
+
+import listenbrainz.db.user as db_user
+import logging
+import sqlalchemy
+
+app = create_app()
+influx = init_influx_connection(logging, {
+            'REDIS_HOST': app.config['REDIS_HOST'],
+            'REDIS_PORT': app.config['REDIS_PORT'],
+            'REDIS_NAMESPACE': app.config['REDIS_NAMESPACE'],
+            'INFLUX_HOST': app.config['INFLUX_HOST'],
+            'INFLUX_PORT': app.config['INFLUX_PORT'],
+            'INFLUX_DB_NAME': app.config['INFLUX_DB_NAME'],
+        })
+
+
+def fix_username_for_exceptions():
+    with db.engine.connect() as connection:
+        # 2106 - Fée Deuspi
+        connection.execute(sqlalchemy.text("""
+            UPDATE "user"
+               SET musicbrainz_id = 'Fée Deuspi'
+             WHERE id = 2106
+            """))
+
+        # 243 - ClæpsHydra
+        connection.execute(sqlalchemy.text("""
+            UPDATE "user"
+               SET musicbrainz_id = 'ClæpsHydra'
```

Comment:
LOL

Reply:
just realized, we would need to fix the influx measurement name for these users too. :/

Influx doesn't have an easy way to rename measurements (and there's a lot of listens)

https://listenbrainz.org/user/Cl%EF%BF%BDpsHydra
https://listenbrainz.org/user/F%EF%BF%BDe%20Deuspi

Maybe I should just write the musicbrainz user id's of these two users by hand...
Then, once we allow renaming user accounts, their names would get fixed automatically.